README.md

```diff
@@ -25,13 +25,17 @@ I discovered HCCB, among other formats, through [Specifications For Popular 2D B
 
 ## Getting started
 
-You need to have [Elm](http://elm-lang.org/) 0.18 installed on your machine.
+Install [Elm](https://github.com/elm/compiler/blob/master/installers/linux/README.md) in your $PATH.
 
-Compile the samples in this project with:
+Compile Hccb
 
-    cd examples; make
+    elm make elm make src/Hccb.elm
 
-Then view it:
+Compile an example
+
+    elm make examples/Simple/Simple.elm
+
+Then view the example in <http://localhost:8000/examples/Simple/index.html> with:
```

Comment:
I don't think the angle brackets are necessary for markdown to know this is a link

Reply:
done